controllers/ExtensionController.php

```diff
@@ -359,7 +359,6 @@ public function actionFiles($id)
 
         return $this->render('files', [
             'model' => $model,
-            'files' => $model->downloads,
```

Comment:
Why?

Reply:
View 'files' do not use the "files" parameter